### PR TITLE
Jon/tags

### PR DESCRIPTION
**Pull request by jonborchardt**
preliminary work to before working on merging for fork

* added tags
* updated spacing values

### CODE REVIEW

File: components/Tag.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    &.ant-tag-blue {
+        ${tagStyles.blue};
+    }
+    &.ant-tag-geekblue {
+        ${tagStyles.geekblue};
+    }
+    &.ant-tag-purple {
+        ${tagStyles.purple};
+    }
+`;
```
Comment (codeviking) [inline components/Tag.tsx]: What's the advantage of splitting out `tagStyles`? Could it just be inlined here (less indirection in this case might be easier to follow)?

File: theme/spacing.ts [hunk 2]
```diff
@@ -48,8 +48,10 @@ export const spacing = {
     md: Spacing.fromPixels('md', 16),
     lg: Spacing.fromPixels('lg', 24),
     xl: Spacing.fromPixels('xl', 36),
-    xxl: Spacing.fromPixels('xxl', 48),
-    xxxl: Spacing.fromPixels('xxxl', 64),
+    xxl: Spacing.fromPixels('xxl (depricated, use xl2)', 48),
```
Comment (codeviking) [inline theme/spacing.ts]:
```suggestion
    xxl: Spacing.fromPixels('xxl (deprecated, use xl2)', 48),
```

File: theme/spacing.ts [hunk 3]
```diff
@@ -48,8 +48,10 @@ export const spacing = {
...
+    xl2: Spacing.fromPixels('xl2', 48),
+    xxxl: Spacing.fromPixels('xxxl (depricated, use xl3)', 64),
```
Comment (codeviking) [inline theme/spacing.ts]:
```suggestion
    xxxl: Spacing.fromPixels('xxxl (deprecated, use xl3)', 64),
```

File: components/Tag.tsx (hunk 1 above)
Comment (jonborchardt) [reply components/Tag.tsx]: ahh... where i copied them from, the style gets reused for styling tags in dropdowns (not copied over yet)

**Pull request merged**